docs/api.rst

```diff
@@ -610,11 +627,19 @@ section generates a new *URL* instance.
       URL('http://example.com/path?a=b&c=d')
       >>> URL('http://example.com/path?a=b').update_query('c=d&c=f')
       URL('http://example.com/path?a=b&c=d&c=f')
+      >>> URL('http://example.com/path?a=b')\
+      ...     .update_query([("c[]", "d"), ("c[]", "e")])
+      URL('http://example.com/path?a=b&c[]=d&c[]=e')
 
    .. versionchanged:: 1.0
 
       All multiple key/value pairs are applied to the multi-dictionary.
 
+   .. versionchanged:: 1.4.3
+
+      The ``'[]'`` characters are not escaped in the keys of the query
+      because some endpoints expect those to be present when specifying arrays.
```

Comment:
Do you mind also referencing an RFC document that specifies this?

Reply:
I do not have any RFC on this, I only wrote this based on what you said in https://github.com/aio-libs/yarl/pull/443#discussion_r415751144.

> web-browsers use `varname[]` format for arrays